DataFormats/DeepFormats/interface/DeepFlavourTagInfo.h

```diff
@@ -0,0 +1,37 @@
+#ifndef DataFormats_DeepFormats_DeepFlavourTagInfo_h
+#define DataFormats_DeepFormats_DeepFlavourTagInfo_h
+
+#include "DataFormats/BTauReco/interface/BaseTagInfo.h"
+#include "DataFormats/DeepFormats/interface/DeepFlavourFeatures.h"
+
+#include "DataFormats/PatCandidates/interface/Jet.h"
+
+namespace reco {
+
+template<class Features> class FeaturesTagInfo : public BaseTagInfo {
```

Comment:
if this class is intended to be persisted more than for debugging purposes, there should be a 
``` CMS_CLASS_VERSION(number) ``` added

Reply:
added in https://github.com/cms-sw/cmssw/pull/19893/commits/dcc93323f80ff2a7ea359c87d628bade95915b1d